filesystem/squashfs_superblock.ksy

```diff
@@ -0,0 +1,112 @@
+meta:
+  id: squashfs_superblock
+  title: SquashFS superblock
+  file-extension:
+    - squashfs
+    - snap
+  endian: le
+  license: CC-0
```

Comment:
The key `license` is supposed to contain a valid SPDX expression, as stated in the [KSY Style Guide](https://doc.kaitai.io/ksy_style_guide.html#meta):

> 3. Legal information — `license` — MUST be a valid [SPDX license expression](https://spdx.org/licenses/)

License identifier `CC-0` is not in the [list](https://spdx.org/licenses/), so I supposed you mean the [Creative Commons Zero v1.0 Universal](https://spdx.org/licenses/CC0-1.0.html) (`CC0-1.0`).

```suggestion
  license: CC0-1.0
```